bundles/org.openhab.binding.mqtt.homeassistant/src/main/resources/OH-INF/thing/homeassistant-channels.xml

```diff
@@ -51,24 +67,40 @@
 	<channel-type id="ha-rollershutter">
 		<item-type>Rollershutter</item-type>
 		<label>Rollershutter</label>
+		<tags>
+			<tag>Control</tag>
+			<tag>OpenLevel</tag>
+		</tags>
 		<config-description-ref uri="channel-type:mqtt:ha-channel"/>
 	</channel-type>
 
 	<channel-type id="ha-string">
 		<item-type>String</item-type>
 		<label>String</label>
+		<tags>
```

Comment:
I would rather not tag this at all, until I can implement dynamic tagging based on additional semantics from the discovery data. For example, this could be read only or read/write, might represent a list of modes on a thermostat, or many other arbitrary use cases.

Reply:
Remove just the commented ones. The dimmer and color channels only apply to lights, so should be good as tagged.